libkbfs/kbfs_ops_test.go

```diff
@@ -3939,34 +3939,10 @@ func TestSetExFileSuccess(t *testing.T) {
 	testSetExSuccess(t, File, true)
 }
 
-func TestSetNoExFileSuccess(t *testing.T) {
```

Comment:
rather than remove the tests, can you modify the tests to deduce / take a parameter as to what result to expect? We already condition on symlinks.